src/pylyrics/extract_lyrics.py

```diff
@@ -2,61 +2,11 @@
 # Date: 2022-01-14
 
 import requests
-import os, sys
-import pandas as pd
-import json
-import urllib.parse
+from bs4 import BeautifulSoup
 import re
-import lyricsgenius
-from alive_progress import alive_bar
```

Comment:
Have you added the relevant packages to .toml?
If not, I will be doing this and pushing my toml because I need to be working with your functions